zebra-rpc/src/methods.rs

```diff
@@ -123,7 +123,7 @@ use types::{
     transaction::TransactionObject,
     unified_address::ZListUnifiedReceiversResponse,
     validate_address::ValidateAddressResponse,
-    z_validate_address::{ZValidateAddressResponse, ZValidateAddressType},
+    z_validate_address::{z_validateaddress, ZValidateAddressResponse, ZValidateAddressType},
```

Comment:
```suggestion
    z_validate_address::{z_validateaddress, ZValidateAddressResponse},
```

Please also fix the unused primitives import.

https://github.com/ZcashFoundation/zebra/actions/runs/16324769993/job/46128577190?pr=9706